packages/dds/sequence/src/index.ts

```diff
@@ -11,11 +11,12 @@ export {
     ISerializedInterval,
     IntervalCollection,
     IntervalCollectionIterator,
-    IntervalCollectionValueType,
     IntervalCollectionView,
     SequenceInterval,
-    SequenceIntervalCollectionValueType,
 } from "./intervalCollection";
+export {
+    IValueOpEmitter,
```

Comment:
do these really need to be exported?

Reply:
i don't see anything public and exported that uses it

Reply:
`IntervalCollectionView` constructor for one, not sure if there are more.

Reply:
bummer. i didn't see that one. ok then